docs/detections/alerts-view-details.asciidoc

```diff
@@ -6,111 +6,159 @@
+//link to custom highlighted fields docs
+
+[discrete]
+[[visualizations-section]]
+== Visualizations 
+
+The Visualizations section is located on the **Overview** tab in the right panel. It offers a glimpse of the processes that led up to the alert and occurred after it. 
+
+[role="screenshot"]
+image::images/visualizations-section-rp.png[Visualizations section of the Overview tab, 65%]
+
+Click **Visualizations** to display the following sub-sections:
+
+* **Session view preview**: Shows a preview of <<session-view,session view>> data. Click **Session viewer preview** to open the **Session View** tab in Timeline. 
+
+* **Analyzer preview**: Shows a preview of the <<visual-event-analyzer,visual analyzer graph>>. The preview displays up to three levels of the analyzed event's ancestors and up to three levels of the event's descendants and children. The ellipses symbol (**`...`**) indicates the event has more ancestors and descendants to examine. Click **Analyzer preview** to open the **Event Analyzer** tab in Timeline.
+
+[discrete]
+[[insights-section]]
+== Insights 
+
+The Insights section is located on the **Overview** tab in the right panel. It offers different perspectives from which you can assess the alert. 
+
+[role="screenshot"]
+image::images/insights-section-rp.png[Insights section of the Overview tab, 65%]
+
+Click **Insights** to display the following sub-sections:
+
+* <<entities-section>>
+* <<threat-intelligence-section>> 
+* <<correlations-section>>
+* <<prevalence-section>>
 
 [discrete]
-[[alerts-enrich-ti]]
-==== Alerts enriched with threat intelligence
-Alerts can be enriched with contextually relevant threat intelligence that you can use for triaging and investigating alerts. If available, this information appears in the Enriched data section.
+[[entities-section]]
+=== Entities 
 
-When gathering threat intelligence data for an alert, {elastic-sec} queries the alert for indicator matches from the past 30 days. The query looks for the following fields:
+The Entities sub-section shows the user related to the alert and the host on which the alert was generated against. <<host-risk-score,Host>> and <<user-risk-score,user risk>> scores are also available with a https://www.elastic.co/pricing[Platinum subscription] or higher. 
+
+NOTE: User and host risk scores are technical preview features. 
+
+Click **Entities** to expand the **Entities** sub-tab in the left panel. The **Entities** sub-tab provides more information about the user and host, including notable activity on related hosts and users. 
+
+[discrete]
+[[threat-intelligence-section]]
+=== Threat intelligence 
+
+The Threat intelligence sub-section shows contextually relevant threat intelligence that was discovered on the alert. When gathering threat intelligence data for an alert, {elastic-sec} queries the alert for indicator matches from the past 30 days. The query looks for the following fields:
```

Comment:
what do you mean by sub-section here? Do you mean sub-tab (which would be the left panel under Insights) or do you mean the section that we expand in the right panel (what I usually call the overview)

Reply:
Resolving in next batch of updates.